cadquery/cq.py

```diff
@@ -2224,8 +2226,8 @@ def rect(
         :type xLen: float > 0
         :param yLen: length in yDirection ( in workplane coordinates )
         :type yLen: float > 0
-        :param boolean centered: true if the rect is centered on the reference point, false if the
-            lower-left is on the reference point
+        :param centered: should the center or the lower bound be on the reference point? A single
```

Comment:
What?

Reply:
Yeah, that's confusing and I should rewrite it. I should probably use the same text for all these methods as well.

Reply:
This text is going to be used a lot, so I want to make sure it is as concise and clear as possible. Can anyone improve on the following?

```suggestion
        :param centered: If True, the rectangle will be centered around the reference
          point. If False, the corner of the rectangle will be on the reference point and
          it will extend in the positive x and y directions. Can also use a 2-tuple to
          specify centering along each axis.
```

Reply:
I think it sounds fine. "reference point" makes sense (I think) because you can specify a list of points, and not use the workplane center.